package.json

```diff
@@ -225,6 +225,17 @@
           "default": "off",
           "markdownDescription": "Traces the communication between VS Code and the Qute language server in the Output view. Default is `off`.",
           "scope": "window"
+        },
+        "qute.validation.undefined.severity": {
```

Comment:
I think the correct name is:

`quarkus.tools.qute.validation.undefinedVariable.severity`